scripts/regression/setupRegressionTests.js

```diff
@@ -0,0 +1,146 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+// this script setup `presentation-full-stack-tests` for running with older `itwinjs-core` or `appui` versions.
+// it will modify package.json files for `presentation-full-stack-tests` and `presentation-test-utilities` to use specified versions
+
+"use strict";
+
+const { execSync } = require("child_process");
+const fs = require("fs");
+const path = require("path");
+const yargs = require("yargs");
+
+// list of packages from `itwinjs-core`
+const corePackages = [
+  "@itwin/appui-abstract",
+  "@itwin/core-backend",
+  "@itwin/core-bentley",
+  "@itwin/core-common",
+  "@itwin/core-electron",
+  "@itwin/core-frontend",
+  "@itwin/core-geometry",
+  "@itwin/core-i18n",
+  "@itwin/core-orbitgt",
+  "@itwin/core-quantity",
+  "@itwin/ecschema-metadata",
+  "@itwin/ecschema-rpcinterface-common",
+  "@itwin/ecschema-rpcinterface-impl",
+  "@itwin/express-server",
+  "@itwin/presentation-backend",
+  "@itwin/presentation-common",
+  "@itwin/presentation-frontend",
+  "@itwin/webgl-compatibility",
+];
+
+// list of packages from `appui`
+const uiPackages = ["@itwin/core-react", "@itwin/components-react", "@itwin/imodel-components-react"];
```

Comment:
what do you think about reading these from `pnpm-workspace.yaml` instead of having them hardcoded here?

Reply:
Update script to load appui and itwinjs-core package names from `pnpm-workspace.yaml`.